heron/schedulers/src/java/org/apache/heron/scheduler/kubernetes/KubernetesContext.java

```diff
@@ -152,6 +162,36 @@ static String getContainerVolumeMountPath(Config config) {
     return config.getStringValue(HERON_KUBERNETES_CONTAINER_VOLUME_MOUNT_PATH);
   }
 
+  static Set<String> getConfigKeys(Config config, String key) {
```

Comment:
`keyPrefix` might be more explicit than `key`

Reply:
I like the suggestion. I wasn't quite sure how to convey that it's a prefix.